smithy-typescript-codegen/src/test/java/software/amazon/smithy/typescript/codegen/RuntimeConfigGeneratorTest.java

```diff
@@ -22,14 +26,37 @@ public void expandsRuntimeConfigFile() {
         List<TypeScriptIntegration> integrations = new ArrayList<>();
         integrations.add(new TypeScriptIntegration() {
             @Override
-            public void addRuntimeConfigValues(
+            public Map<String, Consumer<TypeScriptWriter>> getRuntimeConfigWriters(
+                    TypeScriptSettings settings,
+                    Model model,
+                    SymbolProvider symbolProvider,
+                    LanguageTarget target
+            ) {
+                Map<String, Consumer<TypeScriptWriter>> config = new HashMap<>();
+                config.put("syn", writer -> {
```

Comment:
Can this write and assert a second value to prove out that values persist when others are overwritten?